src/System.Reflection/tests/ParameterInfoTests.cs

```diff
@@ -148,7 +148,7 @@ public void Attributes(Type type, string name, int index, ParameterAttributes[]
             ParameterInfo parameterInfo = GetParameterInfo(type, name, index);
             foreach (ParameterAttributes attribute in expected)
             {
-                Assert.True(parameterInfo.Attributes.HasFlag(attribute));
+                Assert.True((parameterInfo.Attributes & attribute) != 0);
```

Comment:
Yeah, I was going to leave this out, but figured I might as well just update everything in the repo :)